pkg/authn/multikeychain_test.go

```diff
@@ -70,16 +70,33 @@ func TestMultiKeychain(t *testing.T) {
 			fixedKeychain{regOne: three, regTwo: two},
 		),
 		want: Anonymous,
+	}, {
+		name: "match multiple keychains, take the first",
+		reg:  regOne,
+		kc: NewMultiKeychain(
+			fixedKeychain{regOne: one},
+			fixedKeychain{regOne: three, regTwo: two},
+			fixedKeychain{regOne: two},
+		),
+		want: one, // TODO: test that this actually has one,three,two internall and can return them all if asked.
```

Comment:
What more would it take to do this TODO?

Reply:
deleted, i just copy from what u right, I think it's enough to add ut for 
> Could we have a test for a multi-keychain that has some good and some bad (and some expired and needing refreshing) tokens, like we have in base_test.go?